iocore/eventsystem/UnixEventProcessor.cc

```diff
@@ -32,399 +32,229 @@
 #include "ts/ink_defs.h"
 #include "ts/hugepages.h"
 
+#include <algorithm>
+
 /// Global singleton.
 class EventProcessor eventProcessor;
 
-class ThreadAffinityInitializer : public Continuation
```

Comment:
I don't need an opinion now (with bigger fish to fry) ... but I still don't know what you think.

Chris: <blah blah blah... I could toss all this work I think is awesome into this big garbage bag ....>

Phil: That sounds great

:->.  I'll leave it  till a later discussion then.